projects/example-app/src/app/auth/reducers/auth.reducer.ts

```diff
@@ -14,14 +14,14 @@ export function reducer(
   action: AuthApiActions.AuthApiActionsUnion | AuthActions.AuthActionsUnion
 ): State {
   switch (action.type) {
-    case AuthApiActions.AuthApiActionTypes.LoginSuccess: {
+    case AuthApiActions.loginSuccess.type: {
```

Comment:
Already discussed, so just leaving as a note that `AuthApiActions.loginSuccess` for the case would be preferred but we need the action type.